### ISSUE TITLE
hykes-provision error updates /etc/hosts

### ISSUE THREAD

**Issue by travismcchesney** [cloud-elements/hykes-provisioner#64]
**Description**
Calling `hykes-provision` from a directory that does not contain a valid blueprint will update /etc/hosts with an error response.

**Steps to Reproduce**
1. From any non-blueprint directory, run `hykes-provision create do --dns=hosts --consolidated --specs=small`
2. Enter the password if prompted.

**Expected Results**
- The command should fail with an error, and nothing else is changed

**Actual Results**
- The command fails with an error
- The `/etc/hosts/` file is updated with:

```
error
Unable to find a droplet named ''.  api. console. logs.
```

**Comment by rockymadden** [comment]
Yeah, I've encountered this as well. It was on my mental todo list, which slips from my radar from time-to-time. I believe all hykes projects suffer from this (i.e. odd things happen if you don't use from within a blueprint). Going to add this issue explicitly to those others as well.
